docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: "3"
+
+services:
+
+  php-7.1:
+    build:
+      context: .
+      dockerfile: Dockerfile
```

Comment:
useless (and so for other services)

Reply:
+1 for @mremi 
https://blog.packagist.com/php-versions-stats-2018-2-edition/